cargo-shuttle/src/config.rs

```diff
@@ -294,8 +294,26 @@ impl RequestContext {
     }
 
     pub fn get_local_config(
-        project_args: &ProjectArgs,
+        project_args: &mut ProjectArgs
     ) -> Result<Config<LocalConfigManager, ProjectConfig>> {
+        let root_directory_path = find_root_directory(&project_args.working_directory);
+
+        // Iterates through the directories until it finds the Cargo.toml file (project root) and returns the path
+        pub fn find_root_directory(dir: &Path) -> Option<PathBuf> {
+            for ancestor in dir.ancestors() {
+                if ancestor.join("Cargo.toml").exists() {
+                    return Some(ancestor.to_path_buf());
+                }
+            }
+            None
+        }
+
+        if let Some(working_directory) = root_directory_path {
+            project_args.working_directory = working_directory;
+        } else {
+            panic!("Shuttle.toml not found.");
```

Comment:
```suggestion
            return Err(anyhow!("could not locate the root of a cargo project. Are you inside a cargo project? You can also use `--working-directory` to locate your cargo project."));
```
This should make the error message a bit friendlier.

Reply:
Error message updated.